titan-web-client/src/modules/roster/branches/index.js

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { withRouter } from 'react-router-dom';
+import PageHeader from '../../../layouts/dashboard/components/PageHeader/PageHeader';
+import PageHeaderTitle from '../../../layouts/dashboard/components/PageHeader/PageHeaderTitle';
+import { ContentBlock } from '../../../components/block/ContentBlock';
+import BranchCardGrid from '../components/BranchCardGrid';
+import OrganizationsService from '../../../http/OrganizationsService';
+
+class BranchesScene extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.organizationHttpService = new OrganizationsService();
+    this.state = {
+      branches: [
+        {
+          name: 'Coast Guard',
+          member_count: 15,
+          avatar_url: 'https://i1.wp.com/alidade-mer.com/wp-content/uploads/2014/01/coast-guard-logo-1.jpg'
+        },
+        {
+          name: 'Navy',
+          member_count: 15,
+          avatar_url: 'https://lh6.googleusercontent.com/zHJt3iJJBOF7YjiM5pq856vnrNfuhdAcmPNdYiXj3PmGuLqBbD1IqUKWcNfEGhwUScHILSRqtZmS36tbpFLm7DDTrRiWhCAEZRRQu_gecrYw9HvniyQHQQWiNMyAGcYWxuQA4F3R'
+        },
+        {
+          name: 'Air Force',
+          member_count: 8,
+          avatar_url: 'https://lh5.googleusercontent.com/HrFxHitCXCf8-P6Us5lRkvbf6s4VrrxRBlH2SLKy5OIg99VRi9ewVdhcFaFFDftZm86-PH_mD_ryX7_qx0qxX0nILd75rZkXiamLDPDf09dmVz9FU1o3NNAkPHSpAvuHa9c0WiVA'
+        }
+      ]
+    };
+  }
+
+  componentDidMount () {
+    this.organizationHttpService.getAll()
+      .then((res) => {
+        this.setState({
+          branches: res.data
+        });
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
Add a `FIXME` doc above this console log so we know to add better error handling later.

```
// FIXME update UI to indicate error state.
console.log(err);
```